resources.html

```diff
@@ -2,17 +2,17 @@
 permalink: /resources/
 layout: default
 suffixes:
-    - AndroidStudio,
-    - IDLE,
-    - IntelliJ,
-    - VisualStudio,
-    - VSCode,
-    - Coolors,
-    - CodePen,
-    - FontAwesome,
-    - GoogleFonts,
-    - Repl,
-    - Us
+- AndroidStudio,
```

Comment:
The linter has removed the indentation in the yaml at the top of the resources.html file. We need the indentation so the yaml parses correctly.